### PR TITLE
Create Harvard-de-montfort-university.csl

### PR DESCRIPTION
**Pull request by ALRichards**
This is the De Montfort University version of the Harvard referencing system which we would like to be uploaded to the style repository.

### CODE REVIEW

**Comment by rmzelle** [comment]
Thanks. Could you please make the following changes?

* Change the file name to "harvard-de-montfort-university.csl" (lowercase "h")
* Change the title to `<title>Harvard - De Montfort University</title>`
* Set the style locale to British English (`default-locale="en-GB"` on `<style>`; see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale)
* Add a "template" link, and delete Julian Onions as author. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link
* If at all possible, add a "documentation" link to an online or printed style guide. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link

File: harvard-de-montfort-university.csl [hunk 1]
```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - De Montfort University</title>
+    <id>http://www.zotero.org/styles/harvard-de-montfort-university</id>
+    <link href="http://www.zotero.org/styles/harvard-de-montfort-university" rel="self"/>
+    <link href="http://www.library.dmu.ac.uk/Images/Selfstudy/Harvard.pdf" rel="documentation"/>
+    <info>
+     <link href="http://www.zotero.org/styles/harvard1" rel="template"/>
+   </info>
```
Comment (rmzelle) [inline harvard-de-montfort-university.csl]: @ALRichards, our instructions include the `<info>` and `<info/>` tags just to show you where to add the template link, and you don't actually need to add those. The only thing you needed to add is the `<link .../>` element. Can you remove the info tags on the lines before and after?

**Comment by ALRichards** [comment]
Thank you rmzelle for your help in getting this into shape

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**